patch-package
--- a/node_modules/grunt-mocha/phantomjs/bridge.js
+++ b/node_modules/grunt-mocha/phantomjs/bridge.js
@@ -40,6 +40,7 @@
 
       if (test) {
         data.title = test.title;
+        data.titlePath = test.titlePath();
         data.fullTitle = test.fullTitle();
         data.state = test.state;
         data.duration = test.duration;
--- a/node_modules/grunt-mocha/tasks/mocha.js
+++ b/node_modules/grunt-mocha/tasks/mocha.js
@@ -30,13 +30,15 @@ module.exports = function(grunt) {
   var reporter;
 
   // Growl is optional
-  var growl;
+  var growl = function(){};
+  /*
   try {
     growl = require('growl');
   } catch(e) {
     growl = function(){};
     grunt.verbose.write('Growl not found, \'npm install growl\' for Growl support');
   }
+  */
 
   // Get an asset file, local to the root of the project.
   var asset = path.join.bind(null, __dirname, '..');
@@ -48,7 +50,7 @@ module.exports = function(grunt) {
 
     // Hook on Phantomjs Mocha reporter events.
     phantomjs.on('mocha.*', function(test) {
-      var name, fullTitle, slow, err;
+      var name, titlePath, fullTitle, slow, err;
       var evt = this.event.replace('mocha.', '');
 
       if (evt === 'end') {
@@ -57,6 +59,11 @@ module.exports = function(grunt) {
 
       // Expand test values (and façace the Mocha test object)
       if (test) {
+        titlePath = test.titlePath;
+        test.titlePath = function() {
+          return titlePath;
+        };
+
         fullTitle = test.fullTitle;
         test.fullTitle = function() {
           return fullTitle;
